README.md

```diff
@@ -1 +1,98 @@
-# first-mobile-potfoilo
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [First Mobile Porfolio] <a name="about-project"></a>
+
+**[first-mobile-porfolio]** is a project about my qualifications in programming
+
+## 🛠 Built With <a name="built-with"></a>
+   - HTML & CSS
+   - Linters
+   - Git
+   - Github
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+ Clone the repo named first-mobile-portfolio
+
+### Setup
+
+Clone this repository to your desired folder:
+  git@github.com:fauziyahzeeyah/first-mobile-potfoilo.git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Fauziyah**
+
+- GitHub: [@zeeyah](https://github.com/fauziyahzeeyah)
+- Twitter: [@FauziyahZeeyah](https://twitter.com/FauziyahZeeyah?t=5x4tHwdJG62bloZ3QehDcw&s=08)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```

Comment:
- [ ] [OPTIONAL] Kindly remove comments, guides and all unused sections in your README file to make it  clean and professional :+1: :+1: